pkg/util/cert/certutil.go

```diff
@@ -315,3 +316,115 @@ func GenerateSigningRequestInPemFormat(priv *rsa.PrivateKey, commonName string,
 	signingReq := buf.Bytes()
 	return signingReq, err
 }
+
+func isSSLCertInJKS(data map[string][]byte) error {
```

Comment:
**Recommended**: We **SHOULD** only use `is...()` functions for predicates (functions with boolean return values), we can use check.../validate/etc. other function names for optional error returning functions for clarity purposes.